.github/CODEOWNERS

```diff
@@ -546,15 +546,15 @@
 /core/controllers/custom_landing_pages*.py @jameesjohn @vojtechjelinek
```

Comment:
Can I be added to this page, the Contact page, Error pages, Get Started page, Teach page, and Thanks page?